### DISCUSSION TITLE
Website component options

### DISCUSSION THREAD

**Discussion by BrunoBernardino** [serverless/serverless#10647] [Q&A] [answered]
I've been using the `website` serverless component for a while, and a couple of months ago I asked in the respective repo ( https://github.com/serverless-components/website/issues/88 ) for help/suggestions on implementing an improvement. Is there a better place to do so, or should I look into another alternative? The repo hasn't received an update from any of the maintainers for almost a year.

Thank you!

**Comment by mnapoli** [comment, marked as answer]
Hi, sorry about the lack of response, I've posted a comment in https://github.com/serverless-components/website/issues/88

**Reply by BrunoBernardino** [reply]
Thank you!
